test/test.cpp

```diff
@@ -462,7 +462,114 @@ int do_test_pow(uint64_t aStartPos, int aLabelsCount, unsigned aDiff, unsigned a
 						printf("%u hashes, %u h/s, solution at %u\n", (uint32_t)hashes_computed, (uint32_t)hashes_per_sec, (uint32_t)idx_solution);
 						if (-1 != cpu_id) {
 							uint8_t hash[32];
-							scryptPositions(cpu_id, s_id, idx_solution, idx_solution, 256, s_salt, SPACEMESH_API_COMPUTE_LEAFS, hash, 512, 1, 1, NULL, NULL, &hashes_computed, &hashes_per_sec);
+							scryptPositions(cpu_id, s_id, idx_solution, idx_solution, 256, s_salt, SPACEMESH_API_COMPUTE_POW, hash, 512, 1, 1, NULL, NULL, &hashes_computed, &hashes_per_sec);
+							printf("id: ");
+							print_hex32(s_id);
+							printf("\n");
+							printf("salt: ");
+							print_hex32(s_salt);
+							printf("\n");
+							printf("D: ");
+							print_hex32(D);
+							printf("\n");
+							printf("H: ");
+							print_hex32(hash);
+							printf("\n");
```

Comment:
Why print so much in tests?

Reply:
I just copied an existing test and modified it to test for the specific change I made.

I can remove those print statement for the new test if you want.